R/utilities-simulation.R

```diff
@@ -226,9 +254,10 @@ runParallelPopulationSimulation <- function(structureSet,
 #' @import ospsuite
 updateSimulationIndividualParameters <- function(simulation, individualParameters = NULL) {
   if (is.null(individualParameters)) {
-    return()
+    return(TRUE)
   }
   ospsuite::setParameterValuesByPath(parameterPaths = individualParameters$paths, values = individualParameters$values, simulation = simulation)

```

Comment:
Why the return values here? This only makes sense if you try catch the call to ospsuite right? Otherwise it's always true or throws an exception

Reply:
updateSimulationIndividualParameters is used in updateIndividualParametersOnCores to verify and log that the parameters were loaded on each core.